static/app/components/group/tagFacets/index.tsx

```diff
@@ -173,9 +173,14 @@ export default function TagFacets({
     return formatted as Record<string, GroupTagResponseItem>;
   }, [data, tagFormatter, isStatisticalDetector]);
 
-  const topTagKeys = tagKeys.filter(tagKey => Object.keys(tagsData).includes(tagKey));
+  // filter out replayId since we no longer want to
+  // display this on issue or trace details
+  const topTagKeys = tagKeys
+    .filter(tagKey => Object.keys(tagsData).includes(tagKey))
+    .filter(f => f !== 'replayId');
```

Comment:
nit: doing two filters is somewhat inefficient as you have to loop through it twice. you should be able to do it in the line above.

Reply:
ah good point, kept them separate as i thought it'd be more readable/understandable but i can refactor